buildSrc/src/main/java/org/elasticsearch/gradle/test/rest/RestTestUtil.java

```diff
@@ -55,32 +53,6 @@ static RestIntegTestTask setupTask(Project project, String sourceSetName) {
     static void setupRunnerTask(Project project, RestIntegTestTask testTask, SourceSet sourceSet) {
         testTask.setTestClassesDirs(sourceSet.getOutput().getClassesDirs());
         testTask.setClasspath(sourceSet.getRuntimeClasspath());
-
```

Comment:
Why is this no longer needed?

Reply:
The test task depending on the module is handled within testclusters now. The plugin extension dependsOn is handled inside PluginBuildPlugin.